src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/provisioner/main.py

```diff
@@ -83,10 +79,37 @@ def create_or_update_account(org_session, account, adf_role_name, account_id=Non
     org_session.create_account_alias(account.alias, role)
 
     if account.tags:
-        LOGGER.info(f'Ensuring tags exist for account {account_id}: {account.tags}')
+        LOGGER.info(
+            f'Ensuring tags exist for account {account_id}: {account.tags}')
         org_session.create_account_tags(account_id, account.tags)
 
 
+@tenacity.retry(wait=tenacity.wait_fixed(5), stop=tenacity.stop_after_attempt(60))
+def get_all_regions(ec2_client):
+    try:
+        all_regions = [
+            region['RegionName']
+            for region in ec2_client.describe_regions(
+                AllRegions=False,
+                Filters=[
+                    {
+                        'Name': 'opt-in-status',
+                        'Values': [
+                            'opt-in-not-required',
+                        ]
+                    }async def funcname(parameter_list):
```

Comment:
this seems like its in the incorrect place

Reply:
Ouch! Seems I've unwittingly pasted something that hasn't to be there. Apologies. I'm closing this PR and I'll open another one.